recipe/install_base_plugins.sh

```diff
@@ -2,6 +2,9 @@
 
 pushd plugins_base
 
+# Maybe do this only for cos7?
```

Comment:
This can be removed now too. Some helpful debug statements you can use right here could be:

```
pkg-config --variable pc_path pkg-config
pkg-config --libs --cflags gl
```
The first will spit out the current directory config for pkg-config and the second will print out if the gl lib can be found.